lib/code_corps_web/controllers/task_list_controller.ex

```diff
@@ -28,7 +28,7 @@ defmodule CodeCorpsWeb.TaskListController do
     end
   end
 
-  @preloads [:tasks]
+  @preloads [{:tasks, [:comments, :github_pull_request, :task_skills, :user_task]}]
```

Comment:
This is using nested preloads.  I am not sure if we should just add the whole list or not.  Was just adding them based on the feedback from the tests.  @begedin thoughts?

Reply:
ah these are the ones marked with `identifier: always`

Reply:
@JoshSmith would you also `include` the tasks relationships?

Reply:
The real answer here is that it doesn't much matter what I think, but what the performance implications are. Testing this out in the browser with Scout running and doing more precise benchmarking will combined yield a better answer.